### ISSUE TITLE
MySQL Source - Failed to properly convert data value of type DATE (CDC mode)

### ISSUE THREAD

**Issue by alexchouraki** [airbytehq/airbyte#10399]
## Environment
Is this your first time deploying Airbyte: No
OS Version / Instance: Ubuntu 20.04 on AWS EC2
Deployment: Docker
Airbyte Version: 0.35.27-alpha
Source name: MySQL (0.5.4)
Destination: Snowflake (0.4.8)
Severity: Medium
Step where error happened: Sync job

## Current Behavior
Hello!
I’m having issues with the source-mysql  Airbyte connector, with a CDC configuration. 
Incremental sync fails with this type of error log:
```

2022-02-14 14:15:11 source > 2022-02-14 14:15:11 ERROR i.d.r.TableSchemaBuilder(lambda$createValueGenerator$5):269 - Failed to properly convert data value for 'prestashopprod.ps_product.available_date' of type DATE for row [1, 0, 0, 13, 1, 97, 0, 0, [], [54, 54, 48, 48, 52, 50, 55, 49, 51, 54, 55, 52], 0.000000, 0, 1, 499.000000, 0.000000, [], 0.000000, 0.00, [67, 82, 88, 48, 48], [], [], 0.000000, 0.000000, 0.000000, 1.900000, 2, 0, 0, 0, 0, 0, 404, 0, 1, null, new, 1, 0, both, 0, 0, 0, 0, 2017-05-30T17:06:36.000+0000, 2020-04-23T10:32:53.000+0000, 0, 3]:
2022-02-14 14:15:11 source > org.apache.kafka.connect.errors.DataException: Invalid Java object for schema type STRING: class java.lang.Integer for field: "available_date"
2022-02-14 14:15:11 source > 	at org.apache.kafka.connect.data.ConnectSchema.validateValue(ConnectSchema.java:245) ~[connect-api-2.6.1.jar:?]
2022-02-14 14:15:11 source > 	at org.apache.kafka.connect.data.Struct.put(Struct.java:216) ~[connect-api-2.6.1.jar:?]
2022-02-14 14:15:11 source > 	at io.debezium.relational.TableSchemaBuilder.lambda$createValueGenerator$5(TableSchemaBuilder.java:265) ~[debezium-core-1.4.2.Final.jar:1.4.2.Final]
even though, when looking at the table, the row actually seems to have okay values...
```

To sum up : https://github.com/airbytehq/airbyte/issues/6884 this issue that I was encountering previously before seems to be fixed, but now I’m having the cousin of this one https://github.com/airbytehq/airbyte/issues/9118... 
Not sure if this would fix it, but any plans to upgrade Debezium at some point?

## Expected Behavior
Ideally, a successful incremental sync...

## Steps to Reproduce
1. Launch a sync job with MySQL, configured with a CDC,  and tables including dates

**Comment by grishick** [comment]
Seems related to https://github.com/airbytehq/airbyte/issues/14628

**Comment by grishick** [comment]
@alexchouraki we've made a lot of fixes to data type conversion and upgraded Debezium to 1.9 in MySQL Source connector. Could you re-try the sync?

**Comment by alexchouraki** [comment]
Seems to be working properly, thank you!
